payroll/models.py

```diff
@@ -187,6 +187,28 @@ def employee_salaries(self):
         return employee_salaries
 
 
+class EmployerAlias(AliasModel):
+    employer = models.ForeignKey(
+        'Employer',
+        related_name='population',
+        on_delete=models.CASCADE
+    )
+
+    def clean(self):
+        super().clean()
+        try:
+            if self.is_department:
+                type(self).objects.get(
```

Comment:
This is a really clever use of `type()`!

Reply:
thanks! saw it here first: https://stackoverflow.com/questions/2064480/django-best-practice-way-to-get-model-from-an-instance-of-that-model/6759498#6759498